index.html

```diff
@@ -28,344 +28,245 @@
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="img/github.png" alt="github logo" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="img/medium.png" alt="medium logo" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="img/twitter.png" alt="twitter logo" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="img/angellist.png" alt="angellist logo" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="popup" data-popup-id="myPopup">
+      <div class="popup-content">
+        <div class="title popup-title">
+          <h2 class="popup-h2">Tonic</h2>
+          <i class="fas fa-times close-popup" data-dismiss-popup></i>
+          <div class="canopy popup-canopy">
+            <h4>Canopy</h4>
+            <ul>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <img src="./img/snapshootportfolio.png" alt="my first work" />
+        </div>
+        <div class="window-details">
+          <p class="window-descript">
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it 1960s with the releaLorem Ipsum is simply dummy
+            text of the printing and typesetting industry. Lorem Ipsum has been
+            the industry's standard dummy text ever since the 1500s, when a
+            galley of type and scrambled it 1960s with the releawn printer took
+            a galley of type and scrambled it 1960s with the releaLorem Ipsum is
+            simply dummy text of the printing and typesetting industry. Lorem
+            Ipsum has been the industry's standard dummy text ever since the
+            1500s, when an unknown printer took a galley of type and scrambled
+            it 1960s with the relea
+          </p>
+          <div class="aside-clm">
+            <ul class="window-languages">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+              <li>github</li>
+            </ul>
+            <div class="window-btn-links">
+              <a class="btn-live link" target="_blank" href="#"
+                >See live <img src="./img/popup2.svg" class="up" alt="popup"
+              /></a>
+              <a class="btn-code link" target="_blank" href="#"
+                >See source
+                <img src="./img/github1.svg" class="up" alt="github"
+              /></a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>

```

Comment:
- [ ] In this particular project, the whole projects section along with the modals should be created dynamically with Javascript. 🤔  The base projects section is already done (good job! ), 👍🏻 but the models are still not being created with Javascript. Please use Javascript to dynamically render your modals. 😄 

- You could reuse the code that you already have.
- That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `data` array.

![image](https://user-images.githubusercontent.com/57421823/152509546-f41b40b6-dc88-4e3f-9ab4-7bee886e1f52.png)